src/testRunners/jasmine/__tests__/customMatchers-test.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @emails oncall+jsinfra
+ */
+'use strict';
+
+jest.autoMockOff();
+
+const CALLED_AT_LEAST_ONCE = 'function expected to be called at least once';
+const SHOULD_NOT_BE_CALLED = 'function expected not to be called was called';
+const NOT_EXPECTED_VALUES = (
+  'function was never called with the expected values.\n' +
+  'Expected:\n[1,{},"Error"]'
+);
+const NOT_EXPECTED_VALUES_LAST_TIME = (
+  'function wasn\'t called with the expected values.\n' +
+  'Expected:\n[1,{},"Error"]\n' +
+  'Actual:\n[1,{},""]'
+);
+const SHOULD_NOT_HAVE_CALLED_WITH = (
+  'function shouldn\'t have called with\n[1,{},""]'
+);
+
+const getMockedFunctionWithExpectationResult = (expectationResult) => {
+  const mockedFunction = jest.genMockFunction();
+  const expectation = expect(mockedFunction);
+  if (expectation.addExpectationResult) {
+    expectation.addExpectationResult = expectation.not.addExpectationResult = expectationResult;
+  } else {
+    // skip this tests for jasmine 1 for now by returning a mocked expectation
+    return {
+      toBeCalled() {},
+      toBeCalledWith() {},
+      lastCalledWith() {},
+      not: {
+        toBeCalled() {},
+        toBeCalledWith() {},
+        lastCalledWith() {},
+      },
+      function: mockedFunction,
+    };
+  }
+  expectation.function = mockedFunction;
+  return expectation;
+};
+
+describe('jasmine', () => {
+
+  describe('when using a getter for a custom matcher message', () => {
+
+    it('should not get message if test pass', () => {
```

Comment:
in recent years I and some other people have started to drop the `should` from specs. It makes everything nicer and a bit shorter, like:
```js
it(`doesn't access message if test passes`)
```

would you mind updating this test file to do this too? :)